src/astro/files/locations/google/gcs.py

```diff
@@ -1,9 +1,10 @@
 from typing import Dict, List
 from urllib.parse import urlparse, urlunparse
 
+from airflow.providers.google.cloud.hooks.gcs import GCSHook
```

Comment:
@kaxil would this not cause import errors if the provider isn't installed?

Reply:
Since this module isn't imported directly from anywhere we should go good in my opinion.

This module is only imported from https://github.com/astronomer/astro-sdk/blob/main/src/astro/files/locations/__init__.py#L16-L27

Reply:
ohhh interesting ok.